README.rst

```diff
@@ -97,22 +97,17 @@ familiar with the `PyAnsys Developer's Guide`_.
 Test
 ====
 
-.. note::
-
-   The Docker container referenced in the first option is not yet publicly available.
-
-Set the environment variable ``ANSYSLMD_LICENSE_FILE`` to configure the licensing or pass it
-as an argument to the pytest call: ``--license-server=1055@mylicenseserver``.
-
-There are three ways to run the PyDPF Composites tests, depending on how the DPF
+There are different ways to run the PyDPF Composites tests, depending on how the DPF
 server is started.
 
 #.  Run tests with a Docker container:
 
+    Follow the steps in `Getting The DPF Server Docker Image`_ to get
+    and run the dpf docker image. Run the tests with the following command
+
     .. code:: bash
 
-        docker pull ghcr.io/pyansys/pydpf-composites:latest
-        pytest .
+        pytest . --port 50052
 
 
 #.  Run tests with a DPF server started from the Ansys installer. The Ansys version must
```

Comment:
Should we update this to "2024 R1 or later", since the current version will not be compatible with 23R2?